cirq-web/MANIFEST.in

```diff
@@ -0,0 +1 @@
+include cirq_ts/dist/*.bundle.js
```

Comment:
we don't need this file, MANIFEST.in is only for source distributions and we don't publish source distros.

Reply:
Sounds good, I fixed by just adding to package_data

Reply:
Yes, package_data determines what the wheel file (binary distribution) contains.
I guess you still need to delete this file! :)